readthedocs/settings/base.py

```diff
@@ -313,6 +313,7 @@ def INSTALLED_APPS(self):  # noqa
 
     # Misc application settings
     GLOBAL_ANALYTICS_CODE = 'UA-17997319-1'
+    USE_PROMOS = False
```

Comment:
Thoughts on doing this dynamically by checking for the `donate` app? That's how it's done other places (settings, urls) for the ad stuff.

Reply:
I don't have a position on this since I don't konw what the `donate` app really involves and if it would fit correctly here. Besdies, wasn't the variable/app renamed to `readthedocs-ext`?

If it fits correctly, I'm 👍

On the other hand, this way with just a setting it's very easy to understand, override and follow. _Maybe_ base this into an installed app adds a confusing layer when it's not necessary and we don't have many benefits.

Reply:
If the promo code doesn't work without `readthedocs-ext` installed, then doing dynamically based on the package being imported seems like a good choice.